reporter/heap.go

```diff
@@ -0,0 +1,34 @@
+package reporter
+
+// An IntHeap is a max-heap of ints
+type IntHeap []int
+
+// Len returns the number of items in the heap
+func (h IntHeap) Len() int {
+	return len(h)
+}
+
+// Less tells whether elements at position j is less than the one at position i
```

Comment:
We should add a comment here about why we are using `>` instead of `<` operator.

The comment on line 11 could be updated as follows:
```
// Less tells whether elements at position i is **greater** than the one at position j. 
// NOTE: We want Pop to give us the highest priority value so we use greater than here.
```